### DISCUSSION TITLE
Add Motion Sensor Detection Alerts

### DISCUSSION THREAD

**Discussion by SuperDTF** [SecKatie/ha-wyzeapi#651] [Ideas]
We need this - Imagine having your cameras send signals to activate your switches... Any idea how this can be done?

**Comment by hooonza** [comment]
There is a hack you can use. Make a Wyze automation which disables notifications at the cam in the Wyze app with a motion detected. This is a trigger you can detect with this integratoom in HA. And make a HA automation which turns the Wyze notifications on again, on this trigger, with others .
